mega-zep-frontend/src/main/angular/frontend/src/app/modules/shared/components/datepicker/datepicker.component.html

```diff
@@ -1,5 +1,5 @@
 <mat-form-field>
-  <input matInput [matDatepicker]="picker" [max]="today" [value]="selectedDate ? selectedDate : null" disabled>
+  <input [matDatepicker]="picker" [max]="today" [value]="selectedDate" disabled matInput>
```

Comment:
Directly saving selectedDate is the same as the Elvis expression. If selectedDate is null, then the value will be null, if selectedDate has a value, then the value is saved.